core/src/main/java/by/ilyin/core/entity/CustomJWT.java

```diff
@@ -0,0 +1,42 @@
+package by.ilyin.core.entity;
+
+import lombok.*;
+
+import javax.persistence.*;
+import java.time.LocalDateTime;
+
+@Getter
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
+@Entity
+@ToString
+@Table(name = "user_tokens")
+public class CustomJWT {
```

Comment:
@YauhenIlyin Why did you write overhead here?

Reply:
because we have to store the tokens somewhere. because logout and refresh are present in the system, some tokens become invalid